### DISCUSSION TITLE
Files missing from pull request

### DISCUSSION THREAD

**Discussion by davaya** [community/community#8714] [Pull Requests]
When creating a pull request from the web interface, GitHub creates a "patch-1" branch in the user's repo.  The edit screen (for example, for README.md) says "Attach files by dragging & dropping, selecting, or pasting them", and when you do it uploads them and creates links to them in the edited file.

But when you commit, the files never show up.  They did successfully upload to GitHub and can be downloaded from the link address (e.g., https://github.com/OrgName/RepoName/files/1234567/FileName.pdf), but the "files" folder never shows up in RepoName, and the PR shows only 1 file updated (the README).

This feels like a bug, but if not how do you get files to show up in the commit?

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
